amplify/src/main/java/com/github/stkent/amplify/tracking/Amplify.java

```diff
@@ -72,27 +72,42 @@
     private String packageName;
     private String feedbackEmailAddress;
 
-    public static Amplify get(@NonNull final Context context) {
-        return get(context, new Logger());
+    public static Amplify initSharedInstance(@NonNull final Application app) {
```

Comment:
I like this change, seems closer to the installer type pattern. Was this change to allow you to obtain an Amplify instance without a context? Seems slightly strange to have a split init vs get method but still allow mutation of the shared instance.